prow/git/git.go

```diff
@@ -488,3 +488,18 @@ func (i *Repo) ShowRef(commitlike string) (string, error) {
 	}
 	return strings.TrimSpace(string(out)), nil
 }
+
+// FetchFromRemote runs a git fetch command for the specified remote.
+// Pass the destination as org/repo format since the actual URI will be constructed
+// using the existing credentials and will always use the https protocol.
+func (r *Repo) FetchFromRemote(orgRepo, branch string) error {
+	r.logger.Infof("Fetching from '%s (branch: %s)'.", orgRepo, branch)
+	remoteURI := fmt.Sprintf("https://%s:%s@%s/%s", r.user, r.pass, r.host, orgRepo)
```

Comment:
Interesting, didn't know this works without the `.git` suffix
/lgtm
/approve

Reply:
/hold

Reply:
This will break if user or password are unset, no? Do we generally assume that they are always set?

Reply:
Adding a check for this won't hurt. Addressing...

Reply:
Addressed